package.json

```diff
@@ -4,7 +4,9 @@
   "description": "React components for Wonder Blocks design system.",
   "main": "index.js",
   "scripts": {
+    "install": "for d in packages/*/ ; do (cd \"$d\" && npm install); done",
```

Comment:
this should go away using yarn workspaces. :)

Reply:
yup... I made a task for this https://github.com/Khan/wonder-blocks/issues/2.

Reply:
wonderful!